### ISSUE TITLE
feature: Update the MySQL kernel version to the latest official version

### ISSUE THREAD

**Issue by Nliver** [stoneatom/stonedb#1288]
**Is your feature request related to a problem? Please describe.**

No issue has been assigned at the moment, we need to evaluate whether we should follow up the latest version first~

**Describe the solution you'd like**

At present, the MySQL kernel of StoneDB is mysql-5.7.36, but the community version of MySQL official website has been iterated to mysql-5.7.41, between which, there are several minor versions. In fact, while we continue to optimize StoneDB's own code, we should be aware that updates to the latest version of the MySQL community may fix some important bugs and security vulnerabilities, as well as performance optimizations. Therefore, we should evaluate whether the MySQL kernel in StoneDB should be updated to the latest version of the official website.

[MySQL-5.7 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/)

![image](https://user-images.githubusercontent.com/34496563/218362238-3cdf64c8-fffe-4039-87d7-589621e8ac51.png)

**Describe alternatives you've considered**

**Additional context**

**Comment by RingsC** [comment]
OK, yes, we will catch up the upstream.

**Comment by Nliver** [comment]
Of course, the latest version may not be the best, but our current version of the kernel is a year away from the official version, which is objectively enough time to test the minor versions
